app/assets/javascripts/spree/frontend/affirm_checkout.js

```diff
@@ -0,0 +1,74 @@
+(function(){
+  /* only include this setup once */
+  if (!window.AffirmPaymentMethods) {
```

Comment:
If you only need this to run once, we can do that with `affirm.ui.ready()` which we're already using below.